src/Microsoft.OData.Core/ODataOutputContext.cs

```diff
@@ -686,6 +690,30 @@ protected virtual void Dispose(bool disposing)
         {
         }
 
+#if NETCOREAPP3_1_OR_GREATER
+        /// <summary>
+        /// IAsyncDisposable.DisposeAsync() implementation to asynchronously cleanup unmanaged resources of the context.
+        /// </summary>
+        /// <returns>A task representing the asynchronous disposal of the context.</returns>
+        public async ValueTask DisposeAsync()
+        {
+            await this.DisposeAsyncCore().ConfigureAwait(false);
+
+            this.Dispose(false);

```

Comment:
Again, is it okay to call synchronous dispose in asynchronous DisposeAsync?

Reply:
Same case as https://github.com/OData/odata.net/pull/2683/files#r1228957103